web/client/plugins/MapSearch.jsx

```diff
@@ -1,88 +0,0 @@
-/*
- * Copyright 2016, GeoSolutions Sas.
- * All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree.
-*/
-import { connect } from 'react-redux';
-
-import { get } from 'lodash';
-
-import {
-    loadMaps,
-    mapsSearchTextChanged,
```

Comment:
please go around codebase to clean up also actions reducers or selectors no longer needed and remove them  like this mapsSearchTextChanged